tasks/ebpf.py

```diff
@@ -0,0 +1,125 @@
+try:
+    from termcolor import colored
+except ImportError:
+    colored = None
+import contextlib
+import json
+import sys
+
+from invoke import task
+
+try:
+    from tabulate import tabulate
+except ImportError:
+    tabulate = None
+
+from .system_probe import build_object_files, is_root
+
+headers = [
+    "Program",
+    "Stack Usage",
+    "Instructions Processed",
+    "Instructions Processed limit",
+    "Verification time",
+    "Max States per Instruction",
+    "Peak States",
+    "Total States",
+]
+
+
+def tabulate_stats(stats):
+    table = list()
+    for key, value in stats.items():
+        row = list()
+        row.append(key)
+        row.append(value["stack_usage"])
+        row.append(value["instruction_processed"])
+        row.append(value["limit"])
+        row.append(value["verification_time"])
+        row.append(value["max_states_per_insn"])
+        row.append(value["peak_states"])
+        row.append(value["total_states"])
+        table.append(row)
+
+    return tabulate(table, headers=headers, tablefmt="github")
+
+
+def colored_diff(val1, val2):
+    if val1 <= val2:
+        return colored(val1 - val2, "green")
+    return colored(val1 - val2, "red")
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏‏If the `from termcolor import colored` gets an `ImportError`, `colored` will be `None` and this will fail with a `TypeError: 'NoneType' object is not callable`. I think you should either make `termcolor` a mandatory import, or take this case into account !

Reply:
I want to avoid adding another dependency in the buildimages for a task which does not run in the CI.
I will print a more meaningful message when termcolor is not installed.